### ISSUE TITLE
bug: escaped newline in `echo` does not display properly

### ISSUE THREAD

**Issue by NoamBechhofer** [t3-oss/create-t3-app#1804]
### Provide environment information

  System:
    OS: Windows 11 10.0.22631
    CPU: (12) x64 AMD Ryzen 5 5500U with Radeon Graphics
    Memory: 661.51 MB / 7.33 GB
  Binaries:
    Node: 20.11.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 9.6.7 - ~\AppData\Roaming\npm\npm.CMD
  "ct3aMetadata": {
    "initVersion": "7.30.0"
  }

### Describe the bug

When running `./start-database.sh` for a project setup with postgresql, if you do not have Docker installed, an error message is printed followed by an exit.
This error message contains a rogue `\n` that is not being interpreted by `echo`.

### Reproduction repo

https://github.com/NoamBechhofer/t3-issue

### To reproduce

In an environment without docker installed, create a new project that uses prisma/postgresql. Run `./star-database.sh`. An error will be printed: "Docker is not installed. Please install docker and try again.\nDocker install guide: https://docs.docker.com/engine/install/". Note the \n is not escaped.

**Comment by NoamBechhofer** [comment]
I'll fix this and open a PR

**Comment by kakeWasTaken** [comment]
Issue has already been fixed in this commit (#1816). The problem was caused by the echo command on line 15 of the "start-database.sh" script not containing the "-e" option that allows the interpretation of backslash escapes sequences like "\n". This issue should already be resolved in new projects. However, if it is bothering you in an existing project, simply change line 15 of the "start-database.sh" script from 
```shell
echo "Docker is not installed. Please install docker and try again.\nDocker install guide: https://docs.docker.com/engine/install/"
```
to 
```shell
echo -e "Docker is not installed. Please install docker and try again.\nDocker install guide: https://docs.docker.com/engine/install/"
```
